File: event-archive/src/main/java/com/omnibus/archive/repository/RuleUpdateArchiveRepository.java
package com.omnibus.archive.repository;

import com.omnibus.archive.model.RuleUpdateByDay;
import com.omnibus.archive.model.RuleUpdateByDayKey;
import org.springframework.data.cassandra.repository.ReactiveCassandraRepository;

public interface RuleUpdateArchiveRepository
    extends ReactiveCassandraRepository<RuleUpdateByDay, RuleUpdateByDayKey> {
}

File: event-archive/src/main/kotlin/com/omnibus/archive/repository/RuleUpdateArchiveRepository.kt
package com.omnibus.archive.repository

import com.omnibus.archive.model.RuleUpdateByDay
import com.omnibus.archive.model.RuleUpdateByDayKey
import org.springframework.data.cassandra.repository.ReactiveCassandraRepository

interface RuleUpdateArchiveRepository : ReactiveCassandraRepository<RuleUpdateByDay, RuleUpdateByDayKey>

